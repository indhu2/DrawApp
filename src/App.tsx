import React from 'react'
import Home from './screens/Home';
import {Provider} from 'react-redux';
import { persistor, store } from './redux/store';
import { PersistGate } from 'redux-persist/integration/react';

const App = () => {
  return (
    <Provider store={store}>
      <PersistGate loading={null} persistor={persistor}>
        <Home />
      </PersistGate>
    </Provider>
  );
}

export default App