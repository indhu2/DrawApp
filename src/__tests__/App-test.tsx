/**
 * @format
 */

import 'react-native';
import * as React from "react";
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import App from '../App';

test('App renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});