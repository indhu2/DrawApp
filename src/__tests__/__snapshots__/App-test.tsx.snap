// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaView
    edges={
      Array [
        "top",
        "right",
        "left",
      ]
    }
    style={
      Object {
        "backgroundColor": "#FCF5F3",
      }
    }
  />
  <View
    style={
      Object {
        "alignContent": "center",
        "alignItems": "center",
        "backgroundColor": "#F93333",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "alignItems": "center",
          "height": 1334,
          "justifyContent": "center",
          "width": 750,
        }
      }
    >
      <Image
        resizeMode="stretch"
        source={
          Object {
            "testUri": "../../../src/assets/images/png/splash_bg.png",
          }
        }
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": 1334,
              "width": 750,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "position": "absolute",
            "zIndex": 1,
          }
        }
      >
        <svg
          data-file-name="SvgLogo"
        />
        <Text
          align="left"
          bold={false}
          color="#F93333"
          fontFamily="Nunito-Regular"
          fontSize={25.142857142857142}
          fontWeight={null}
          lineHeight={0}
          marginBottom={0}
          marginLeft={0}
          marginRight={0}
          marginTop={0}
          style={
            Array [
              Object {
                "color": "#F93333",
                "fontFamily": "Nunito-Regular",
                "fontSize": 25.142857142857142,
                "textAlign": "left",
              },
              Object {
                "marginVertical": 31.428571428571427,
                "textAlign": "center",
              },
            ]
          }
          width={null}
        >
          Education
           
          <Text
            align="left"
            bold={false}
            color="#353535"
            fontFamily="Nunito-Regular"
            fontSize={25.142857142857142}
            fontWeight={null}
            lineHeight={0}
            marginBottom={0}
            marginLeft={0}
            marginRight={0}
            marginTop={0}
            style={
              Array [
                Object {
                  "color": "#353535",
                  "fontFamily": "Nunito-Regular",
                  "fontSize": 25.142857142857142,
                  "textAlign": "left",
                },
                Object {
                  "marginVertical": 31.428571428571427,
                  "textAlign": "center",
                },
              ]
            }
            width={null}
          >
            without boundaries
          </Text>
        </Text>
      </View>
      <View
        style={
          Object {
            "bottom": 80,
            "position": "absolute",
          }
        }
      >
        <ActivityIndicator
          animating={true}
          color="#F93333"
          size="large"
        />
      </View>
    </View>
  </View>
</View>
`;
